code/game/objects/effects/spiderwebs.dm

```diff
@@ -61,15 +61,15 @@
 		return
 	if(sealed)
 		return FALSE
-	if(istype(mover, /mob/living/simple_animal/hostile/giant_spider))
+	if(isspider(mover))
 		return TRUE
 	else if(isliving(mover))
 		if(istype(mover.pulledby, /mob/living/simple_animal/hostile/giant_spider))
```

Comment:
you missed one

Reply:
actually this doesnt take into account istypes that check a var rather than having the direct mob. Few more examples are sentient disease's ``apply_innate_effecs``, payment component's ``attempt_charge``, ``user_try_print_id`` in _production.dm, and others, but I'd rather that be in a separate PR because I don't want to re-read over all this